app/src/main/java/com/frinder/frinder/activity/DiscoverActivity.java

```diff
@@ -163,56 +166,44 @@ public void onClick(View view) {
     }
 
     private void getSelectedInterest(int position) {
-        //Note: This is to support filtering by multiple interests
-        //But the clicked interests will remain in original position and not jump to beginning of list
-                /*Interest interestClicked = interests.get(position);
-                String interestClickedDBValue = interestClicked.getDBValue();
-
-                if (filterInterestList.contains(interestClickedDBValue)) {
-                    filterInterestList.remove(interestClickedDBValue);
-                    interestClicked.setSelected(false);
-                }
-                else {
-                    filterInterestList.add(interestClickedDBValue);
-                    interestClicked.setSelected(true);
-                }
-
-                interestsAdapter.notifyItemChanged(position);
-
-                Log.d(TAG, "Interests picked = " + filterInterestList.toString());
-                */
-
         Interest interestClicked = interests.get(position);
         String interestClickedDBValue = interestClicked.getDBValue();
+        int originalArrayPosition = interestClicked.getOrigArrayPosition();
 
+        Log.d(TAG, "\nINTERESTPICK previousInterestClickedPosition = " + previousInterestClickedPosition
```

Comment:
The initial "\n" is not really required.

Reply:
I had too many Logs and was trying to space them out on the screen to find out what was happening. Should I comment out the logs when I push to Github master?